homeassistant/components/homeassistant/repairs.py

```diff
@@ -40,7 +40,7 @@ async def async_step_ignore(
     ) -> FlowResult:
         """Handle the ignore step of a fix flow."""
         ir.async_get(self.hass).async_ignore(
-            DOMAIN, f"integration_not_found.{self.domain}", True
+            HOMEASSISTANT_DOMAIN, f"integration_not_found.{self.domain}", True
```

Comment:
This is a wierd and incorrect change. For this specific location, it is actually the `DOMAIN`, as this is the `homeassistant` integration component.

Reply:
You're right - I've adjusted to import DOMAIN from local `const.py`